plugins/org.orbisgis.groovyeditor/test/org/orbisgis/ui/editors/groovy/ClassPathHandlerTest.java

```diff
@@ -0,0 +1,82 @@
+package org.orbisgis.ui.editors.groovy;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ClassPathHandlerTest {
+
+    private List<URL> urls;
+    ClassPathHandler classPathHandler;
+
+    @BeforeEach
+    void beforeEach(){
+        urls = new ArrayList<URL>();
+        classPathHandler = new ClassPathHandler();
+    }
+
+    @AfterEach
+    void afterEach() throws MalformedURLException {
+        if (classPathHandler.getUrls().size() > 0){
+            for(int i=0; i< classPathHandler.getUrls().size(); i++){
```

Comment:
As you have an indexed for loop, you don't need to check if the urls list is empty or not.